examples/eval_model.py

```diff
@@ -30,8 +31,17 @@ def setup_args():
 def eval_model(parser, printargs=True):
     random.seed(42)
     opt = parser.parse_args(print_args=False)
+    # check to make sure the model file exists
+    nomodel = Flase
+    if opt.get('model_file') is None or not os.path.isfile(opt['model_file']):
```

Comment:
i think if model file is set and file doesn't exist, you can bail immediately.
it's only the 'not set' case you have to check if it has a load function or so?